.storybook/recipes/CalendarCard/CalendarCard.module.css

```diff
@@ -0,0 +1,70 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #CALENDAR CARD
+\*------------------------------------*/
+
+/**
+ * Card recipe for displaying calendar data with a colored left border (color dependent on variant)
+ * 1) Brand variant (default): brand colored left border
+ * 2) Revise variant
+ * 3) Success variant
+ */
+.calendar-card {
+  width: 16rem;
+  background-color: var(--eds-theme-color-background-neutral-default-hover);
+}
+
+.calendar-card--brand {
+  /* 1 */
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-brand-primary-strong);
+}
+
+.calendar-card--revise {
+  /* 2 */
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-grade-revise-strong);
+}
+
+.calendar-card--success {
+  /* 3 */
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-utility-success-strong);
+}
+
+.calendar-card__body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: var(--eds-size-1);
+}
+
+.calendar-card__title {
+  @mixin eds-typography-preset-007-bold;
```

Comment:
Sorry, I'm having trouble understanding the question. Why can't we use `@mixin eds-theme-typography-label-md;`?

>but we only have label-md as the 007-bold value

Isn't the text bold in both the design and the preset?

Reply:
If we want all the tier 2 type presets available for use by the `Heading` component, we could add them to a `typography.module.css` file that both `Heading` and `Text` use. i.e.
```
// typography.module.css
.typography--label-md {
  @mixin eds-theme-typography-label-md;
}

// usage
<Heading as="h3" size="label-md" />
<Text size="label-md" />
```

is that what you're thinking @ifrost1 ?

Reply:
@dierat @anniehu4 Thanks for this. 

> Why can't we use @mixin eds-theme-typography-label-md;

We can. The issue lies in other designers using this for future cases and this being baked into this specific recipe.

> is that what you're thinking @ifrost1 ?

That is what I'm thinking. In an ideal world, we'd never have to style a `Heading` outside of the `Heading` component's CSS. That being said, if you don't want that in the `Heading` component, I understand having it as a recipe-specific thing. That being said, who is stopping designers from using that in other patterns? It would be nice to have the `Heading` handle typography preset changes. I prefer Annie's suggestion just because it gives options if other designs come out with this treatment simply because designers will design and I'd imagine this may get used again at some point.

Reply:
Another option is to create a `title-xs` token that would be more associated with headings. `label-md` is associated with form field labels and that may differ in terms of multiple themes using this stuff.